.travis.yml

```diff
@@ -77,6 +77,7 @@ before_script:
        -DCeres_DIR="${DEPS_INSTALL_PATH}/share/Ceres" \
        -DEIGEN_INCLUDE_DIR_HINTS="${DEPS_INSTALL_PATH}" \
        -DOpenMVG_USE_CCTAG=OFF \
+       -DPOPSIFT_ROOT=$POPSIFT_INSTALL \
```

Comment:
need to install popsift and cuda too
https://github.com/poparteu/openMVG/blob/popsift_squash/ci/install-dependencies.sh#L20